phone_book/celery.py

```diff
@@ -0,0 +1,14 @@
+from celery import Celery
+import os
+from phone_book import settings
+
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'phone_book.settings')
+app = Celery('phone_book')
+
+app.config_from_object('django.conf:settings')
+app.autodiscover_tasks(lambda: settings.INSTALLED_APPS)
+
+
+@app.task(bind=True)
+def debug_task(self):
```

Comment:
What is this task doing?

Reply:
get this from celery documentation somehow it is related to the debugging of task but not sure because not enough documentation and comments on this 
**source:** http://docs.celeryproject.org/en/latest/django/first-steps-with-django.html

Reply:
I believe this is just an example task that was used in the documentation. We don't have a need for this in our app.

Reply:
ok got it